### DISCUSSION TITLE
現地参加の定員を設定出来るようにする

### DISCUSSION THREAD

**Discussion by jacopen** [cloudnativedaysjp/dreamkast#1363] [Ideas]
ハイブリッド化をするにあたって、現地参加の定員を設定し、上限に達した場合は登録できないようにする必要がある。

**Comment by jacopen** [comment]
セッションごとの登録者数は、registered_talksから簡単に導き出せる。ただ、現地参加なのかオンライン参加なのかは今のままだと判断がつかないので、何らしかのフラグを追加する必要がある。

現地参加の定員については、
- Talksにカラムを追加して設定
- Tracksにカラムを追加して設定
の2択が考えられる。同一トラックでも定員が変わる可能性があるのなら前者、ないのなら後者。んー

**Comment by oshiro3** [comment]
現地開催を考えると 「track = 部屋」が基本になるので track で持たせてしまってもよさそうです。ただ、track には日付が関係ないので「1日目は平日なので track C は定員30人(小部屋)にしたが2日目は休日で track C も定員100人にする」のようなことが起こりえないかが気がかりです。
これまで原則平日に行っているのでそのようなイレギュラーは発生し難そう & track の調整でなんとかなりそうなので大丈夫かとは思いますが。
また、原則を track に持たせておけば特定の talk で定員を変えたいなどの場合はその時に talk にも定員を設定して優先度を付ければ良さそうかなと思いました。
